libvncserver/websockets.c

```diff
@@ -74,8 +74,7 @@ Connection: Upgrade\r\n\
 Sec-WebSocket-Accept: %s\r\n\
 \r\n"
 
-#define WEBSOCKETS_CLIENT_CONNECT_WAIT_MS 100
-#define WEBSOCKETS_CLIENT_SEND_WAIT_MS 100
+#define WEBSOCKETS_CLIENT_CONNECT_WAIT_MS 500
```

Comment:
In most use cases 500 ms is more than acceptable, but, generally speaking, we should maybe put all of these constants to a config.h.in file so that a developer could set them to their liking at compile time.

We should probably address this in a future PR, my comment is more to remember.